rest-server/src/components/auth/authController.js

```diff
@@ -0,0 +1,33 @@
+import User from '../user/userController';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import { error } from '../../lib/log';
+
+export default class Auth {
```

Comment:
I've been mulling this over for the longest time but I don't we should stick to OOD here. Not a lot of need or uses for classes in our controllers. Let's just stick to regular functions